expansion/src/main/resources/parentPublicationFrame.json

```diff
@@ -23,6 +23,12 @@
       "@explicit": true,
       "publicationContext": {
         "@default": "@null"
+      },
+      "publicationInstance": {
+        "@explicit": true,
+        "type": {
+          "@default": "@null"
+        }
```

Comment:
This seems correct, I wonder if we have a default flag in framing that we do not intend?